docs/packages/pkg/xxhash.rst

```diff
@@ -0,0 +1,21 @@
+.. spelling::
+
+    xxhash
+
+.. index::
+  single: crypto ; xxhash
+
+.. _pkg.xxhash:
+
+xxhash
+======
+
+-  `Official <https://github.com/Cyan4973/xxHash>`__
+-  `Hunterized <https://github.com/Warchant/xxHash>`__
```

Comment:
Should we move it to https://github.com/hunter-packages ?
- https://docs.hunter.sh/en/latest/creating-new/patch.html

Reply:
Yes. Should I transfer it to hunter-packages?

Reply:
Created: https://github.com/hunter-packages/xxHash

Please send your patches there. Also, it's better to ask for it before submitting a pull request, see:
- https://docs.hunter.sh/en/latest/creating-new/patch.html